keel-titus-plugin/src/main/kotlin/com/netflix/spinnaker/keel/api/titus/cluster/TitusClusterSpec.kt

```diff
@@ -52,8 +55,12 @@ data class TitusClusterSpec(
   @JsonInclude(JsonInclude.Include.NON_EMPTY)
   val overrides: Map<String, TitusServerGroupSpec> = emptyMap(),
   @JsonIgnore
-  override val deliveryArtifact: DeliveryArtifact? = null,
-  @JsonIgnore
+  val containerProvider: ContainerProvider,
```

Comment:
`TitusClusterSpec` now holds a `ContainerProvider` so we can find artifact references to support the association between resources and artifacts, similar to `ClusterSpec`.